xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -923,10 +939,10 @@
         <Parameter Name="value" Type="System.UInt16" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="source">The read-only span of bytes to read.</param>
+        <param name="value">When the method returns, the value read out of the read-only span of bytes, as big endian.</param>
+        <summary>Reads a <see cref="T:System.UInt16" /> out of a read-only span of bytes as big endian.</summary>
+        <returns>If the span is too small to contain a <see cref="T:System.UInt" />, return <see langword="false" />; otherwise, return <see langword="true" />.</returns>
```

Comment:
This caused the build warning:

```suggestion
        <returns>If the span is too small to contain a <see cref="T:System.UInt16" />, return <see langword="false" />; otherwise, return <see langword="true" />.</returns>
```